app/db.py

```diff
@@ -16,6 +16,11 @@ class Pageloads(Base):
     page_end = Column(DateTime, nullable=True)
 
 
+class Users(Base):
+    __tablename__ = 'users'
+    name = Column(String(), primary_key=True)
```

Comment:
do you need () after string if you are not defining a length?

Reply:
No which means the decision falls to our coding style.  I need to figure out what PEP-8 says

Reply:
PEP-8 does not say what to do. We might want to standardize this ourselves

Reply:
@michardy im ok with either, your call